src/index.js

```diff
@@ -574,6 +566,10 @@ class LocalstackPlugin {
    */
   async reconfigureAWS() {
     if(this.isActive()) {
+      if(this.reconfigured_endpoints){
```

Comment:
nit: we usually use `camelCase` in JavaScript/TypeScript code - could be changed to `this.reconfiguredEndpoints`